_posts/2020-08-26-bazel-3.5.md

```diff
@@ -0,0 +1,68 @@
+---
+layout: posts
+title: "Bazel 3.5"
+authors:
+  - aiuto
+---
+
+[Bazel 3.5](https://github.com/bazelbuild/bazel/releases/tag/3.5.0) has just been released.
+
+Bazel 3.5 is a minor release. It is an incremental update to Bazel 3.4.
+
+## Incompatible changes
+
+- Remove the `--experimental_process_wrapper_wait_fix` flag.
+- Remove the `--experimental_ui_deduplicate` flag.
+- Remove the `--experimental_transparent_compression` flag.
+- Remove the `--experimental_action_args` flag.
+- Remove the `--incompatible_symlinked_sandbox_expands_tree_artifacts_in_runfiles_tree` flag.
+- Bazel now correctly prefers Xcode versions in `/Applications` over
+  Other paths, by default.  This resolves an issue with accidentally
+  picking up an Xcode version from a Time Machine backup or network
+  disk.  If you rely on the old behavior, move your desired Xcode
+  version to `/Applications`.
+
+## New features
+
+- `cquery` now follows aspects when the `--include_aspects` flag is used.
+- Add support to `bazel/crosstool` for building arm64 on macos.
+
+## Important changes
```

Comment:
* Merge "Important features" and "New features" (some things below are actually new features, but the distinction is not useful).
* Rename the section "Other changes" and put it below "Rules" ("Other changes" is the term we used in other announcements).